ch341a_spi.c

```diff
@@ -434,7 +434,7 @@ int ch341a_spi_init(void)
 
 	/* Enable information, warning, and error messages (only). */
 #if LIBUSB_API_VERSION < 0x01000106
```

Comment:
Are you sure this condition is true for your version of libusb?

Reply:
No, I'll check.

Reply:
I'm using flashrom-9.9+r1954. On line 447 of ch341a_spi.c there is a call to libusb_set_debug that causes warnings. I can see in more recent versions of flashrom that you're checking the LIBUSB_API_VERSION so obviously this patch is only useful for older versions.